readme/BREAKING_CHANGES.markdown

```diff
@@ -480,4 +480,22 @@ The `frontend-taglib-clay` module is now using components from
 [`Clay v3`](https://github.com/liferay/clay), which doesn't support the
 previous attributes.
 
----------------------------------------
\ No newline at end of file
+---------------------------------------
+### Handling HTML boolean attributes in tags
+- **Date:** 2021-Feb-18
+- **JIRA Ticket:** [LPS-127832](https://issues.liferay.com/browse/LPS-127832)
+
+#### What changed?
+
+If a boolean attribute is passed to a tag with any other value than
+`true` will be not rendered.
+
+#### Who is affected?
+
+Anyone using tag libraries
+
+#### Why was this change made?
+
+A lot of tags accept dynamic attributes that need to be rendered correctly.
+After reading the [spec](https://html.spec.whatwg.org/#boolean-attribute) on boolean attributes, we think that if one of those attributes
+receives any other value than `true` it shouldn't be rendered.
```

Comment:
I would avoid saying "we think" in a document like this. We're potentially breaking things for customers, so we should present a confident-sounding rationale for doing so:

Suggest:

```
This change is being made for better compliance with [the HTML Standard](https://html.spec.whatwg.org/#boolean-attribute),
which says that "The presence of a boolean attribute on an element represents
the true value, and the absence of the attribute represents the false value."
```